tools/editor/project_manager.cpp

```diff
@@ -1133,18 +1133,20 @@ void ProjectManager::_files_dropped(StringArray p_files, int p_screen) {
 			memdelete(dir);
 		}
 		if (confirm) {
-			multi_scan_ask->set_text(TTR("Are you sure to scan " + itos(multiple_folders.size()) + " folders?"));
+			multi_scan_ask->get_ok()->disconnect("pressed", this, "_scan_multiple_folders");
+			multi_scan_ask->get_ok()->connect("pressed", this, "_scan_multiple_folders", varray(folders));
+			multi_scan_ask->set_text(TTR("Are you sure to scan " + itos(folders.size()) + " folders?"));
```

Comment:
`TTR()` does not support concatenated strings (it will extract only the the first part for the translation).
You should use instead `vformat` to allow using a substitution marker, like that:
```
vformat(TTR("You are about the scan %s folders for existing Godot projects. Do you confirm?"), folders.size())
```
(I also rephrased a bit the sentence ;))
Also with `vformat` you don't need `itos()`, as the `%s` substitution marker forces the conversion of the argument to String (if possible).